assets/csidriveroperators/shared-resource/05_clusterrole.yaml

```
@@ -88,8 +88,15 @@ rules:
   resources:
   - customresourcedefinitions
   verbs:
-  - list
   - create
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions/sharedconfigmaps.sharedresource.openshift.io
+  - customresourcedefinitions/sharedsecrets.sharedresource.openshift.io
+  verbs:
+  - list
+  - get
```

This should use `resourceNames` in the RBAC, and restrict `update`, and `delete`:

```suggestion
- apiGroups:
  - apiextensions.k8s.io
  resources:
  - customresourcedefinitions
  resourceNames:
  - sharedconfigmaps.sharedresource.openshift.io
  - sharedsecrets.sharedresource.openshift.io
  verbs:
  - update
  - delete
```